gems/aws-sdk-core/lib/seahorse/client/async_base.rb

```diff
@@ -4,8 +4,11 @@ module Seahorse
   module Client
     class AsyncBase < Seahorse::Client::Base
 
-      @plugins = PluginList.new
-
+      @plugins = PluginList.new([
+        Plugins::Endpoint,
+        Plugins::H2,
+        Plugins::ResponseTarget
+      ])
```

Comment:
space + comments for api private i think?

Reply:
This is exactly what it was before.  What do you mean?